src/Authentication/Authenticators/JWT/Firebase.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace CodeIgniter\Shield\Authentication\Authenticators\JWT;
+
+use CodeIgniter\Shield\Exceptions\RuntimeException;
+use DomainException;
+use Firebase\JWT\BeforeValidException;
+use Firebase\JWT\ExpiredException;
+use Firebase\JWT\JWT;
+use Firebase\JWT\Key;
+use Firebase\JWT\SignatureInvalidException;
+use InvalidArgumentException;
+use stdClass;
+use UnexpectedValueException;
+
+class Firebase implements JWTDecoderInterface
```

Comment:
I'm not a fan of naming this `Firebase` since that is a product name and this authenticator doesn't have anything to do with Firebase itself, other than the library was provided by them. I'd rename to something like more descriptive like simply `JWT` or similar.